config/database.yml

```diff
@@ -0,0 +1,21 @@
+default: &default
+  adapter: postgresql
```

Comment:
The template won't use a postgresql, we already provide a way to store app/user data using our API.